spec/factories/user.rb

```diff
@@ -2,6 +2,7 @@
   factory :user do
     sequence(:email) { |i| "test#{i}@example.com" }
     password { "pAssword1" }
+    organisation
```

Comment:
Would this not be ` organisation { FactoryBot.create(:organisation) }`?

Reply:
This is shorthand for that. So `organisation` tells FactoryBot to try and look up a factory with that name.

Reply:
Ah I see 😄